docs/generated/packages/react-native.json

```diff
@@ -797,6 +797,10 @@
           "include": {
             "type": "string",
             "description": "A comma-separated list of additional npm packages to include. e.g. `nx sync-deps --include=react-native-gesture-handler,react-native-safe-area-context`"
+          },
+          "exclude": {
+            "type": "string",
```

Comment:
We should convert this into `type: testPathPattern` now. I think it was set to string because parsing args are array wasn't supported before.

Schema would look like this:

```json
"exclude": {
  "type": "array",
  "items": {
    "type": "string"
  },
  "default": []
},
```

Same for `include`... then we don't need to do the `.split(',')` anymore.